Tests/NIOSSLTests/NIOSSLIntegrationTest.swift

```diff
@@ -21,7 +21,7 @@ import CNIOBoringSSL
 import NIO
 @testable import NIOSSL
 import NIOTLS
-import class Foundation.Process
+//import class Foundation.Process
```

Comment:
I think commenting this out is unwise: if we don’t need it, we should delete it.